specification/compute/resource-manager/Microsoft.Compute/preview/2016-04-30-preview/compute.json

```diff
@@ -1892,6 +1892,104 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/guestResourceTypes/software/items":{
```

Comment:
lets make the values of guestResourceTypes as an enum and add the allowed values there.